attribution-graphs-frontend/attribution_graph/cg.css

```
@@ -0,0 +1,772 @@
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+        }
+    }
+
+    .subgraph {
+        overflow: hidden;
+        /* outline: 1px solid #eee; */
+
+        marker{
+            orient: auto;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)